<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Campaign_Wizard</class>
 <widget class="QWizard" name="Form_Campaign_Wizard">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouvelle campagne de tests...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWizardPage" name="first_page_wizard">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nom de la campagne :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="campaign_short_name"/>
    </item>
    <item row="3" column="2">
     <widget class="RecordTextEdit" name="campaign_description"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Description :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Projet :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="project_short_name_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Version :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="project_version_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="second_page_wizard">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QRadioButton" name="select_none_test_indic">
      <property name="text">
       <string>Vide (les tests seront copiés manuellement à partir du projet)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Veuillez sélectionner le modèle de campagne :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="select_specifics_tests_indic">
      <property name="text">
       <string>Tests spécifiques</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QRadioButton" name="select_all_tests_indic">
      <property name="text">
       <string>Complète (tous les tests du projet)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QCheckBox" name="category_indic">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Catégorie</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="priority_indic">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Priorité</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comparison">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Nulle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Faible</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Moyenne</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Elevée</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Critique</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="test_priority_level">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="test_category">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RecordTextEdit</class>
   <extends>QTextEdit</extends>
   <header>gui/components/record_text_edit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>campaign_short_name</tabstop>
  <tabstop>campaign_description</tabstop>
  <tabstop>select_none_test_indic</tabstop>
  <tabstop>select_specifics_tests_indic</tabstop>
  <tabstop>category_indic</tabstop>
  <tabstop>test_category</tabstop>
  <tabstop>priority_indic</tabstop>
  <tabstop>comparison</tabstop>
  <tabstop>select_all_tests_indic</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
